src/vault_backup/main.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+import argparse
+import logging
+
+import context
+
+from backup import backup_secrets
+from lib.logging import setup_logging
+
+
+def parse_args():
+  ''' parse command line args '''
+  parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    description=('Backup Vault secrets')
+  )
+  parser.add_argument(
+    'artsy_env',
+    choices=['staging', 'production'],
+    help='the artsy environment of Vault instance'
+  )
+  parser.add_argument(
+    '--loglevel',
+    choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+    default='INFO',
+    help='log level'
+  )
+  parser.add_argument(
+    '--s3',
+    action='store_true',
+    help='indicates to save backup to S3'
+  )
+  return parser.parse_args()
+
+def parse_env():
+  ''' parse env vars '''
+  vault_host = os.environ.get('VAULT_HOST')
+  vault_port = os.environ.get('VAULT_PORT')
+  kvv2_mount_point = os.environ.get('VAULT_KVV2_MOUNT_POINT')
+
+  # local dir to store exported secrets
+  local_dir = os.environ.get('VAULT_BACKUP_ASCII_LOCAL_DIR')
+
+  s3_bucket = os.environ.get('VAULT_BACKUP_ASCII_S3_BUCKET', '')
+  s3_prefix = os.environ.get('VAULT_BACKUP_ASCII_S3_PREFIX', 'dev')
```

Comment:
It's not clear why these two are configured as ENV vars but the `s3` flag is a cli opt. Perhaps these should be cli opts with defaults?